README.md

```diff
@@ -52,3 +52,24 @@ the new version.
 longer compatible. In this case just fix the config files. Some of them come from 
 Ansible making patches here and there but most of the file should be what you see in the
 repos.
+
+
+# update README: How to run integration tests
+
+Step 1: Install Docker, tavern-ci, pytest
+
+Step 2: Run `copy tests/hubapi/demohub/biothing_studio/config.py.example tests/hubapi/demohub/biothing_studio/config.py`, edit file `config.py` (fill: AWS_ACCESS_KEY, AWS_SECRET_KEY)
+
+Step 3: Run `copy tests/hubapi/demohub/testcases/config.yaml.example tests/hubapi/demohub/testcases/config.yaml`, edit file `config.yaml`
+
+Step 4: Run `copy .env.example .env.local`, edit file `.env.local`  (fill: AWS_ACCESS_KEY, AWS_SECRET_KEY)
```

Comment:
@remoteeng00 please also include the details like what s3 buckets need to be created and what permission is needed for the given AWS_ACCESS_KEY and AWS_SECRET_KEY.

Reply:
It's udpated.